tools/generators/files_and_groups/test/ElementCreator/CreateFileElementTests.swift

```diff
@@ -178,9 +178,9 @@ final class CreateFileElementTests: XCTestCase {
     func test_element_content_lastKnownFileType_file() {
         // Arrange
 
-        let name = "node_name.bazel"
-        let ext = "bazel"
-        let bazelPath = BazelPath("a/bazel/path/node_name.bazel")
+        let name = "node_name.foo"
```

Comment:
nit: Sorry, but can we also add a test that has a known extension, so we verify the non-`"file"` path?

Reply:
Sure, added one for ruby!